.github/workflows/core_build_publish.yml

```diff
@@ -44,10 +77,36 @@ jobs:
       run: dotnet msbuild /t:RunTests /p:Configuration=Release
       working-directory: test
     - name: Pack
-      if: env.IS_MASTER_BUILD == '1'
-      run: dotnet pack --no-build -c Release -o $PWD/packed
-    # - name: Publish
-    #   if: env.IS_MASTER_BUILD == '1'
-    #   run: find packed/ -name '*.nupkg' -exec dotnet nuget push -k "$NUGET_API_KEY" -s 'https://api.nuget.org/v3/index.json' -n true '{}' ';'
-    #   env:
-    #     NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
\ No newline at end of file
+      if: ${{ needs.prepare.outputs.publish_artifacts == '1' }}
+      env:
+        ZIP: ${{ needs.prepare.outputs.artifacts_name }} 
+      run: |
+        dotnet pack --no-build -c Release -o "$PWD/packed"
+        zip -j "$ZIP" "$PWD/packed/*.nupkg"
```

Comment:
And now the glob will also be escaped, resulting in nice `no such file: *.nupkg` :upside_down_face:

Reply:
:angry: